packages/overlay/src/App.tsx

```diff
@@ -81,11 +108,13 @@ export default function App({
     spotlightEventTarget.addEventListener('open', onOpen as EventListener);
     spotlightEventTarget.addEventListener('close', onClose);
     spotlightEventTarget.addEventListener('navigate', onNavigate as EventListener);
+    spotlightEventTarget.addEventListener('sentry:clearEvents', clearEvents as EventListener);
```

Comment:
Any specific reason why `sentry:clearEvents`? 
I would just call it `clearEvents`

Reply:
Right now, it clears out the data from the sidecar and events stored for sentry integration only. so I thought to make it just for sentry integration
that was the reason to add sentry: here 
and to check if sentry integration was added, only show the clear events button.

but I made that generic now, so I am changing this to "clearEvents"